sympy/codegen/ast.py

```diff
@@ -130,12 +130,16 @@
 from sympy.core.basic import Basic
 from sympy.core.compatibility import string_types
 from sympy.core.expr import Expr
-from sympy.core.numbers import Float, Integer
+from sympy.core.numbers import Float, Integer, oo
 from sympy.core.relational import Lt, Le, Ge, Gt
 from sympy.core.sympify import _sympify, sympify, SympifyError
 from sympy.utilities.iterables import iterable
 
 
+inf = float(oo)
```

Comment:
I don't think it's good to have these native Python floats floating around (so to speak). Especially since `oo` is a singleton and often compared with `is`, I think it could cause issues.